File: easy/pro-ballpark-orders/BallparkOrders.rb
tax = 1.07
menu = {Nachos: 6, Pizza: 6, Cheeseburger: 10, Water: 4, Coke: 5}
menu.default = menu[:Coke]

total_cost = gets.split.map {|i| menu[i.to_sym]}.inject(:+)

puts '%.2f' % [total_cost * tax]

File: easy/pro-ballpark-orders/BallparkOrders.py
menu = {'Nachos': 6, 'Pizza': 6, 'Cheeseburger': 10, 'Water': 4, 'Coke': 5}
tax = 1.07

items = input().split()
total_cost = 0

for item in items:
    if item not in menu:
        total_cost += menu['Coke']
    else:
        total_cost += menu[item]

total_cost *= tax

print('%.2f' % total_cost)
